addons/mail/static/src/messaging/component/thread_viewer/thread_viewer.js

```diff
@@ -99,7 +99,8 @@ class ThreadViewer extends Component {
          * scroll positions after everything has been rendered, but OWL doesn't
          * have such an API for the moment.
          */
-        if (messageList) {
+        const shouldAdjust = !('scrollAdjust' in this.props) || this.props.scrollAdjust;
+        if (shouldAdjust && messageList) {
```

Comment:
Here you just want to avoid adjusting scrolling in `MessageList`, so I expect that you still call adjust, but forward props to `MessageList` so that itself ignore adjustment of scroll if it has this prop set.